css/src/components/pagination.css

```diff
@@ -22,28 +21,31 @@
     display: flex;
     align-items: center;
     justify-content: center;
-    padding-left: var(--spacing-xl);
-    padding-right: var(--spacing-xl);
+    padding-left: var(--spacing);
+    padding-right: var(--spacing-2);
 }
 
-.Pagination-pageIndex .Input {
+.Pagination-pageIndex .MetricInput {
     flex-grow: 0;
-    min-width: var(--spacing-3);
-    width: var(--spacing-5);
+    width: var(--spacing-3);
     margin-right: var(--spacing);
+    padding-left: var(--spacing-m);
+    padding-right: var(--spacing-m);
+    display: flex;
+    align-content: center;
 }
 
-.Pagination-pageIndex .Input-input {
+.Pagination-pageIndex .MetricInput-input {
     text-align: center;
 }
 
-.Pagination-pageIndex .Input-input::-webkit-inner-spin-button,
-.Pagination-pageIndex .Input-input::-webkit-outer-spin-button {
+.Pagination-pageIndex .MetricInput-input::-webkit-inner-spin-button,
+.Pagination-pageIndex .MetricInput-input::-webkit-outer-spin-button {
```

Comment:
No need to add this css as we are passing `withArrowIcons: false` to metricInput